wiki/fr/fr-Installation-for-Macintosh.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: wiki
+title: "Installation sous MacOS"
+lang: "fr"
+permalink: "/wiki/Installation-for-Macintosh"
+---
+
+# Installation sous MacOS
+
+Assurez-vous de lire le [guide de démarrage et installation](Getting-Started) avant de commencer.
+
+1. [Téléchargez Jamulus](https://sourceforge.net/projects/llcon/files/latest/download){: target="_blank" rel="noopener noreferrer" .button}
+1. **Décompressez le fichier .zip téléchargé**. Double-cliquez sur le fichier `.zip`, ce qui créera un nouveau dossier du même nom. Le dossier contient le fichier de licence et un fichier `.dmg` contenant l'application.
+1. **Double-cliquez sur le fichier `.dmg`** pour l'ouvrir. L'image s'ouvre et vous verrez deux applications (client et serveur Jamulus).
```

Comment:
OK.

Reply:
I'd be ok with this interpretation. As long as the user knows roughly the same I'd say it's ok.

Reply:
> I'd be ok with this interpretation. As long as the user knows roughly the same I'd say it's ok.

Working together means making compromises...
Even if under MacOS we open a disk image rather than decompressing a file.
In order not to get stuck in a useless debate I adopted the proposed wording.

But I would be happier with the original wording which I think would make more sense for MacOS users.

> We should discuss "interpretation" in another ticket since I think it is worth a discussion.

Yes I think. The culture and languages are different. We can't always stay totaly closed to the original english.

Reply:
If on a Mac, one has to "_open an image_", then it should be written like that in the original English file.

From a non-MacOs user point of view, as long as the original English text it talking about "_unpack_", it translates to "_décompresser_" or "_déballer_" into French. I've choosen "_décompresser_" but feel free to use "_déballer_" if you think it better fit the translation.

That said, if when you click on the file, it writes something like "_Voulez vous ouvrir l'image?_" then we can and should use the original "_L'image s'ouvre_" translation from @gegeweb . I don't own or use a MacOS so can't juge for that. I live it up to a MacOs user.

Reply:
> That said, if when you click on the file, it writes something like "_Voulez vous ouvrir l'image?_" then we can and should use the original "_L'image s'ouvre_" translation from @gegeweb . I don't own or use a MacOS so can't juge for that. I live it up to a MacOs user.

On (the French UI of) MacOS, after double-clicking on the dmg (disk image) or choosing « Ouvrir » on the contextual menu ctrl or right clic you have a message « Ouverture de l'image […] », so it is mounted as a volume and must be ejected as another disk or volume after use. It is not an archive like a zip or tar, it is a disk image.

Reply:
I've planned to open some issue about MacOS, there are several iccomplete or inexact things about MacOS in the original version of the documentation.
I correct by reflex in the translation (except for the compilation page, I didn't touch anything compared to the original, I'll wait to know if it's intentional).

Reply:
Fixed in the last commit.